policies/travel-policy.md

```diff
@@ -206,9 +204,9 @@ Entitlement Table** below:
 ##### Meal Entitlement Table
  | Period | Cost |
  | --- | --- |
- | Full Day | Up to US\$200/day |
- | Partial Day | Up to US\$100/day |
- | Entertainment\* | Full Cost of meal. Alcoholic beverages up to US\$20 per person per meal |
+ | Full Day | Up to US $150 a day |
+ | Partial Day | Up to US $75 a day |
```

Comment:
Usually, you can get per diems ahead of the trip in a cash. Or you can the per diems cash after the trip without any receipts.  You can spend this per diems or you can keep this cash and stay hungry.  In accounting, it is accounted for per diems per country and days. 
So I would say that it would be $450 in total and it doesn't matter if a single day is over the $150 limit.

Reply:
It was intended to be a per day limit, so not accumulative, you get up to the $150 each day but any amount not used doesn't carry over.

If you want to change it to accumulative let me know.

Reply:
> You can spend this per diems or you can keep this cash and stay hungry.

I don't think this was the intent of this policy? At least not the way I understood it.

Reply:
> > You can spend this per diems or you can keep this cash and stay hungry.
> 
> I don't think this was the intent of this policy? At least not the way I understood it.

For all intents and purposes, that is what "per diem" / "allowance" means.  But yeah, this policy seems to rather be about reimbursement, and Tam has made changes accordingly per my comments on the matter this morning, so this review comment is a bit moot at this point.

Reply:
I interpret per diems as per the law if you are a regular employee who has got a business trip. It is as it is. But in our case, we are contractors and we should consider this policy as reimbursement as Richard already noted. So let's keep it as reimbursement for contractors as we do.